content/Unreal Tournament 2004/Models/S/3/2/24b037/sally_[3224b037].yml

```diff
@@ -0,0 +1,115 @@
+--- !<MODEL>
+contentType: "MODEL"
+firstIndex: "2022-05-28 15:45"
+game: "Unreal Tournament 2004"
+name: "Sally"
```

Comment:
Name: Sonic Freedom Fighters Pack.
This pack contains five models: Amy, Bunny, Sonic, Sally and Tails.